runtime/core/frontend/feature_pipeline.cc

```diff
@@ -92,23 +95,61 @@ bool FeaturePipeline::ReadOne(std::vector<float>* feat) {
 
 bool FeaturePipeline::Read(int num_frames,
                            std::vector<std::vector<float>>* feats) {
+  bool b = true;
   feats->clear();
-  std::vector<float> feat;
-  while (feats->size() < num_frames) {
-    if (ReadOne(&feat)) {
-      feats->push_back(std::move(feat));
+  std::vector<std::vector<float>> tmp_feats;
+  std::vector<std::vector<float>> chunk_feats;
```

Comment:
I think the chunk here and the `chunk_size` may be confused.

Reply:
`block_feats` may be better.

Reply:
yes